test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -5575,6 +5575,97 @@ def forward(self, x):
         [np.testing.assert_allclose(ort_out1, ort_out2, atol=1e-7, rtol=0.001) for ort_out1, ort_out2 in
          zip(ort_outs1, ort_outs2)]
 
+    def test_initializer_sequence(self):
+        class MyModule(torch.nn.Module):
+            def __init__(self, input_size, hidden_size, num_classes):
+                super(MyModule, self).__init__()
+                self.fc1 = torch.nn.Linear(input_size, hidden_size)
+                self.relu = torch.nn.ReLU()
+                self.fc2 = torch.nn.Linear(hidden_size, num_classes)
+
+            def forward(self, x):
+                out = self.fc1(x)
+                out = self.relu(out)
+                out = self.fc2(out)
+                return out
+
+        import os
```

Comment:
nit: checkout https://github.com/pytorch/pytorch/blob/0e4e1c5ebdf6e9903f7931d45375873113020c84/test/onnx/test_pytorch_onnx_onnxruntime.py#L43, you can export onnx model to a string, instead of creating a temp file.

Reply:
> nit: checkout
> 
> https://github.com/pytorch/pytorch/blob/0e4e1c5ebdf6e9903f7931d45375873113020c84/test/onnx/test_pytorch_onnx_onnxruntime.py#L43
> 
> , you can export onnx model to a string, instead of creating a temp file.

Sure, it's done.